test/docfx.Test/lsp/LanguageServerTestClient.cs

```diff
@@ -40,7 +42,9 @@ internal class LanguageServerTestClient : ILanguageServerNotificationListener, I
         private int _clientNotificationSent;
         private int _clientNotificationReceived;
         private int _clientNotificationReceivedBeforeSync; // For expectNoNotification
-        private TaskCompletionSource _notificationSync = new TaskCompletionSource();
+        private TaskCompletionSource _notificationSync = new();
+
+        private CancellationToken DefaultCancellationToken => new CancellationTokenSource(30000).Token;
```

Comment:
nit: consider turn this into a method. A property usually implies field semantic, which gives the impression that the cancellation token is shared. Consider a more descriptive name like `TimeOut`